### PR TITLE
add extended readme

### PR DESCRIPTION
**Pull request by denny101603**

### CODE REVIEW

File: readme_developers.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     ```
+     - on Linux, you can use bash script in /scripts/patch_simple_collada_asset.sh
+
+ - download and set custom gradle - follow https://developers.google.com/ar/develop/unity/android-11-build#unity_20193_and_20194 (We recommend "binary-only" version of gradle, since we encountered bugs with "complete" version)
+ - delete folder repository\arcor2_AREditor\Assets\Submodules\TriLib\TriLibSamples
+ - in Unity, go to Project Settings -> XR -> ARCore Extensions and change Android Authentication Strategy from "Keyless" to "Api Key"
+
+### Building
+ - when building for Android: go to Project Settings -> Player -> Other Settings and add "AR_ON" to Scripting Define Symbols
+ - when building for Unity editor (no matter which platform): Remove "AR_ON" from Scripting Define Symbols in Project Settings -> Player -> Other Settings
```
Comment (Kapim) [inline readme_developers.md:52]: When the platform is set to Standalone, AR_ON does not have to be removed and it is ignored. When the platform is set to Android, if you build the .apk or want to use AR Foundation Remote, the AR_ON has to be set, whilst when running play mode without the tablet, AR_ON has to be removed.

**Review by Kapim — CHANGES_REQUESTED** [review]
Please, add a mention that IL2CPP backend is meant mainly for releases, while for faster development cycle the Mono backend is suitable.

File: readme_developers.md (hunk 1 above)
Comment (denny101603) [reply readme_developers.md]: readme updated

**Review by Kapim — APPROVED** [review]
Perfect, thanks :)

**Pull request merged**